data/mysql/schema.sql

```diff
@@ -2239,7 +2239,7 @@ CREATE TABLE `ezuser` (
   `contentobject_id` int(11) NOT NULL DEFAULT '0',
   `email` varchar(150) NOT NULL DEFAULT '',
   `login` varchar(150) NOT NULL DEFAULT '',
-  `password_hash` varchar(50) DEFAULT NULL,
+  `password_hash` varchar(60) DEFAULT NULL,
```

Comment:
This is ok, but if we are to follow suggestions from password-compat we could change it to 255 while on it to have space for future formats. But we should as you already do explicit use `PASSWORD_BCRYPT ` anyway, as we will need to handle migration if `PASSWORD_DEFAULT` suddenly changes.